cmd/virtual-workspaces/command/cmd.go

```diff
@@ -96,7 +98,7 @@ func Run(ctx context.Context, o *options.Options) error {
 	identityConfig, resolveIdentities := bootstrap.NewConfigWithWildcardIdentities(nonIdentityConfig, bootstrap.KcpRootGroupExportNames, bootstrap.KcpRootGroupResourceExportNames, nil)
 	if err := wait.PollImmediateInfiniteWithContext(ctx, time.Millisecond*500, func(ctx context.Context) (bool, error) {
 		if err := resolveIdentities(ctx); err != nil {
-			klog.V(3).Infof("failed to resolve identities, keeping trying: %v", err)
+			logger.V(3).Info(fmt.Sprintf("failed to resolve identities, keeping trying: %v", err))
```

Comment:
this is not yet what we need here (although the linter passed): look into other uses of logger throughout the code-base. Basically if you need `fmt.Sprintf` it's a sign of non-structured logs. Structured logs have pairs of keys and values behind the string. Here `err` is such a value. @stevekuznetsov is the expert here. Maybe he has some good pointers.